### ISSUE TITLE
Accessibility:Contrast Analyser: Notification Templates - PREVIEW: The code in red color is not meeting the standard luminosity ratio equal to 4.5:1 under editing templates.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/monaco-editor#910]
Browser: Any
OS: Windows 10

Steps or JS usage snippet reproducing the issue:
1. Login to "ms.portal.azure.com" with valid credentials
2.Tab to "More Services" from the left navigation panel and search for API Management services
3. Select an existing API Management Service.
4. Tab to "Notification Templates - PREVIEW" under "API Management" and hit enter
5. Tab to a notification in the list and hit enter
6. Tab to the parameters in the right pane and hit enter to add the corresponding parameter in the snippet.
7. Check the contrast ratio for the codes in red color in that code snippet.

Actual Behaviour:
The foreground to background ratio for the codes in red color is 4.0:1.

Expected Behaviour:
The contrast ratio for the codes in red color in the editing snippet should be at least 4.5:1.
![red codes_html_failing_ca](https://user-images.githubusercontent.com/22956130/40974769-3cee0622-68e6-11e8-8202-d70eac2a14c1.png)

**Comment by alexandrudima** [comment]
The default theme for syntax highlighting we ship with is based on Visual Studio's default theme.

Themes can be customized to change these colors. I suggest you reach out to the portal team and ask them to customize the theme if they wish to. We will not change our default theme, as we want it to match Visual Studio's default theme.
